File: C/Sorting/BubbleSorting.c
#include<stdio.h>
void main()
{
    int arr[5],flag=0;
    for(int i=0;i<5;i++)
        scanf("%d",&arr);
    for(int i=0;i<5;i++)
    {
        for(int j=0;j<4;j++)
        {
            if(arr[j]>arr[j+1])
            {
                int temp=arr[j];
                arr[j]=arr[j+1];
                arr[j+1]=temp;
                flag=1;
            }
            if(flag==0)
                break;
        }
    }
    for(int i=0;i<5;i++)
        printf("%d",arr[i]);
}

File: Python/Sorting/BubbleSorting.py
arr=list(map(int,input().split())) #[:5]
flag=True
for i in arr:
    for j in range(0,len(arr)-1):
        if(arr[j]>arr[j+1]):
            arr[j],arr[j+1]=arr[j+1],arr[j]
            flag=False
        if(flag):
            break
print(arr)